native.js

```diff
@@ -3,6 +3,7 @@ const path = require('node:path');
 const { platform, arch, report } = require('node:process');
 
 const isMusl = () => !report.getReport().header.glibcVersionRuntime;
+const msvcOrGnu = report.getReport().header.osName.startsWith('MINGW32_NT');
```

Comment:
I find the name slightly ambiguous, especially since "true" means that it is GNU Windows. But since it is only used in a single place, why not inline it below, then it is completely clear. Also, then it is not evaluated if not needed.
(admittedly, we could also inline isMusl...)